Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView.swift

```diff
@@ -16,6 +16,23 @@ open class ChatChannelAvatarView: _View, ThemeProvider, SwiftUIRepresentable {
     open var content: (channel: ChatChannel?, currentUserId: UserId?) {
         didSet { updateContentIfNeeded() }
     }
+    
+    /// Object responsible for loading images.
+    open lazy var imageLoader: ImageLoading = {
```

Comment:
is there any reason to make all properties lazy? 
can we simplify this code to: 

```Swift
open var imageLoader: ImageLoading = NukeImageLoader()
open var imageMerger: ImageMerging = DefaultImageMerger()
open var imageProcessor: NukeImageProcessor = NukeImageProcessor()
```
I would also assume that we want to do the protocols injections for all dependencies for making FIRST tests, but this can be TBD